azurelinuxagent/ga/exthandlers.py

```diff
@@ -35,36 +35,38 @@
 import azurelinuxagent.common.conf as conf
 import azurelinuxagent.common.logger as logger
 import azurelinuxagent.common.utils.fileutil as fileutil
+from azurelinuxagent.common.protocol.util import NamedSet
```

Comment:
> Since NamedSet doesn't actually have anything to do with protocol, it should be defined elsewhere. Is there a better place for it in one of the common modules?

+1

Reply:
> Since NamedSet doesn't actually have anything to do with protocol, it should be defined elsewhere. Is there a better place for it in one of the common modules?

The apt only place I could find was in future.py (bringing in future python-supported feature to present). Other common utility modules didn't seem to be the correct place. And I didn't want to create a new utility module, which might not be used again, only for this one function.

Reply:
Placed it to `types.py`.